### DISCUSSION TITLE
Possible React Native Support?

### DISCUSSION THREAD

**Discussion by jwaldrip** [panva/jose#173] [Q&A]
I was looking at a ReactNative WebCrypto implementation, and was curious if it could be made to work with jose. https://github.com/kevlened/isomorphic-webcrypto

**Comment by panva** [comment]
> I was looking at a ReactNative WebCrypto implementation, and was curious if it could be made to work with jose. [kevlened/isomorphic-webcrypto](https://github.com/kevlened/isomorphic-webcrypto)

Give it a shot, I'm curious what you find out.
